website/docs/docs/deploy/ci-jobs.md

```diff
@@ -27,8 +38,41 @@ dbt Labs recommends that you create your Slim CI job in a dedicated dbt Cloud [d
+
+1. On your deployment environment page, click **Create Job** > **Continuous Integration Job** to create a new CI job. 
+
+2. Options in the **Job Description** section:
+    - **Job Name** &mdash; Specify the name for this CI job.
+    - **Environment** &mdash; By default, it’s set to the environment you created the CI job from.
+    - **Triggered by pull requests** &mdash; By default, it’s enabled.
+
+3. Options in the **Execution Settings** section:
+    - **Commands** &mdash; By default, it includes the `dbt build --select state:modified+` command. This informs dbt Cloud to build only new or changed models and their downstream dependents. Importantly, state comparison can only happen when there is a deferred job selected to compare state to. Click **Add command** to add more [commands](/docs/deploy/job-commands)  that you want to be invoked when this job runs.
+    - **Compare changes against an environment (Deferral)** &mdash; By default, it’s set to the **Production** environment if you created one. This option allows dbt Cloud to check the state of the code in the PR against the code running in the deferred environment, so as to only check the modified code, instead of building the full table or the entire DAG.
+    - **Generate docs on run** &mdash; Enable this option if you want to [generate project docs](/docs/collaborate/build-and-view-your-docs) when this job runs. This option is disabled by default since doc generation testing isn't typically performed for every CI check.
+
+  <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/create-ci-job.png" width="90%" title="Example of CI Job page in dbt Cloud UI"/>
+
+4. (optional) Options in the **Advanced Settings** section: 
+    - **Environment Variables** &mdash; Define [environment variables](/docs/build/environment-variables) to customize the behavior of your project when this CI job runs. You can specify that a CI job is running in a _Staging_ or _CI_ environment by setting an environment variable and modifying your project code to behave differently, depending on the context.
+    - **Target Name** &mdash; Define the [target name](/docs/build/custom-target-names) to correspond this CI job to the settings in your project. Similar to **Environment Variables**, this option lets you customize the behavior of the project. You can use this option to specify that a CI job is running in a _Staging_ or _CI_ environment by setting the target name and modifying your project code to behave differently, depending on the context. 
+    - **Run Timeout** &mdash; Cancel this CI job if the run time exceeds the timeout value. You can use this option to help ensure that a CI check doesn't consume too many of your warehouse resources.
+    - **dbt Version** &mdash; By default, it’s set to inherit the [dbt version](/docs/dbt-versions/core) from the environment. dbt Labs strongly recommends that you don't change the default setting as mismatched versions can result in unexpected behavior.
```

Comment:
```suggestion
    - **dbt Version** &mdash; By default, it’s set to inherit the [dbt version](/docs/dbt-versions/core) from the environment. dbt Labs strongly recommends that you don't change the default setting by selecting a specific dbt version on the job setting. This option to change the version at the job level is useful only when upgrading a project to the next dbt version, but otherwise, mismatched versions between the environment and job can lead to confusing behavior.
```